src/main/java/me/totalfreedom/totalfreedommod/blocking/MobBlocker.java

```diff
@@ -31,11 +31,13 @@ public void onStop()
     {
     }
 
+    //after careful review by ayunami2000 this code always returns before doing anything
```

Comment:
@Paldiu Up to you to determine if we should make a new PR for this - ideally, each PR should only contain stuff related to the ticket. This change does not fit the scope of the current ticket.

Reply:
Ok, I removed my change to this with my latest commit, but I guess it needs to be put in a Jira ticket to be looked further into? Also I reformatted the rest of it to Allman style with my latest commit as well.